README.md

```diff
@@ -88,6 +88,8 @@ Interested in fixing issues or contributing to Podman Desktop?
 - :checkered_flag: [Read our contributing guide](./CONTRIBUTING.md)
 - :ok_hand: [Review or contribute a pull request](https://github.com/containers/podman-desktop/pulls)
 
+Joining a [community meeting](https://podman-desktop.io/community) is a great way to get involved and help shape the future of Podman Desktop.
+
```

Comment:
The upcoming community meeting is updated at this link https://github.com/podman-desktop/community/blob/main/meetings/README.md#upcoming-community-meeting
If you think pointing to the community page  is better, i am ok with it

Reply:
@rujutashinde this is why I pinged you, link updated